wicket-common/src/update_events.rs

```diff
@@ -219,6 +221,15 @@ pub enum UpdateTerminalError {
         #[source]
         error: anyhow::Error,
     },
+    #[error("Can't update Cosmo")]
+    CosmoHost,
+    #[error("getting SP state failed")]
+    SpGetFailed {
+        #[source]
+        error: gateway_client::Error<gateway_client::types::Error>,
+    },
+    #[error("Unknown host type {0}")]
+    UnknownHost(String),
```

Comment:
Just for consistency with the other errors

```suggestion
    #[error("can't update Cosmo")]
    CosmoHost,
    #[error("getting SP state failed")]
    SpGetFailed {
        #[source]
        error: gateway_client::Error<gateway_client::types::Error>,
    },
    #[error("unknown host type {0}")]
    UnknownHost(String),
```